rules/manager.go

```diff
@@ -281,7 +282,16 @@ func (m *Manager) ApplyConfig(conf *config.Config) {
 	copy(rulesSnapshot, m.rules)
 	m.rules = m.rules[:0]
 
-	if err := m.loadRuleFiles(conf.RuleFiles...); err != nil {
+	var files []string
+	for _, pat := range conf.RuleFiles {
+		fs, err := filepath.Glob(pat)
```

Comment:
Yes, you can glob in the file SD config. But only on the file level. You cannot glob directories.
So yes - you could do `<directory>/prefix_*.rules` – but that's about it.

For file SD the file extensions are fixed on `.json`, `.yml`, and `.yaml`. That's not something I could do for `.rules` without making yet another breaking change.